lib/stat_tracker.rb

```diff
@@ -116,4 +133,15 @@ def average_goals_by_season
       result
     end
   end
+
+# ==================       League Stats Methods      ==================
+
+  def best_offense
+    average_goals_per_game_per_team.max_by { |team, avg| avg }[0].name
+  end
+
+  def lowest_scoring_visitor
+    exclude = "home"
+    average_goals_per_game_per_team(exclude).min_by { |team, avg| avg }[0].name
+  end
 end
```

Comment:
@guardiac thanks for your thoughtfulness on letting this exclude home or away! I can definitely use this in `highest_scoring_home_team`

Reply:
Agreed! I think the optional parameters will come in handy! I'll definitely be able to use it for highest scoring visitor